src/types/webcam.ts

```diff
@@ -1,6 +1,7 @@
 export interface Webcam {
     lat: number;
     lon: number;
+    direction: number;
```

Comment:
I would like the direction to be optional and that when there is no direction, it is not added to the data.